ui/redux/actions/notifications.js

```diff
@@ -192,3 +196,82 @@ export function doDeleteNotification(notificationId: number) {
       });
   };
 }
+
+export function doFetchNotificationSettings() {
+  return (dispatch: Dispatch) => {
+    return Notifications.settings_get()
+      .then(({ data }) => {
+        const notificationSettings = data[0].setting;
+
+        dispatch({
+          type: ACTIONS.NOTIFICATION_SETTINGS_FETCHED,
+          data: { notificationSettings },
+        });
+      })
+      .catch((e) => {
+        dispatch({
+          type: ACTIONS.NOTIFICATION_SETTINGS_FAILED,
+        });
+      });
+  };
+}
+
+export function doToggleNotificationsDisabled() {
+  return (dispatch: Dispatch, getState: GetState) => {
+    const state = getState();
+    const notificationDisabled = selectNotificationDisabled(state);
+    const followerMentions = selectFollowerMentions(state);
+    const followedMentions = selectFollowedMentions(state);
+
+    return Notifications.settings_set({
+      channel_id: '*',
+      channel_name: '*',
```

Comment:
I see a channel selector, but the ID is always the same. I this value valid, or perhaps this is causing the `405`?